### DISCUSSION TITLE
Reputation spike

### DISCUSSION THREAD

**Discussion by Arctomachine** [FakeFishGames/Barotrauma#14206] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

According to mission descriptions, they should have increased reputation by 8+8=16. Instead they gave +128 reputation.
Note that decrease in opposing faction is -5-5=-10, as it should be.
![image](https://github.com/FakeFishGames/Barotrauma/assets/29041820/1f96c9d2-24f9-4a65-8310-7af6d54ab06a)

### Reproduction steps

Happened only once, so no idea how to reproduce.
I accepted missions and started round, then closed game.
Then started game and finished round.

### Bug prevalence

Just once

### Single player or multiplayer?

Single player

### Version

v1.5.8.0 (Summer Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Arctomachine** [comment]
False alarm. Discord user clarified that it is due to multiple security officers having talent for bonus reputation for escort mission.

**Comment by Regalis11** [comment]
Thank you for the report! We have an existing report about the talent issue that seems to be behind these huge reputation gains: https://github.com/FakeFishGames/Barotrauma/discussions/14055
